src/App.js

```diff
@@ -39,6 +39,8 @@ function App() {
           <Route path="/products" element={<Products />} />
```

Comment:
Your routes will go, suggestion: keep them in a separate file

Reply:
Can you give any reference ?